Gemfile

```diff
@@ -39,6 +41,9 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rspec-rails', '~> 3.5'
+  gem 'database_cleaner'
+  gem 'factory_girl_rails'
```

Comment:
[OPTIONAL]
factory_girll is deprecated. Please try and use [factory_bot](https://github.com/thoughtbot/factory_bot) in the future.